<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSlider" name="joint1_slider">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>160</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="joint2_slider">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>190</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="joint3_slider">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>220</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QGraphicsView" name="view">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>451</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="link1_spinbox">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>20</y>
      <width>141</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="link2_spinbox">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>50</y>
      <width>141</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="link3_spinbox">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>80</y>
      <width>141</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="joint1Angle">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>160</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="joint2Angle">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>190</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="joint3Angle">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>220</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="draw_btn">
    <property name="geometry">
     <rect>
      <x>458</x>
      <y>120</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate Arm</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reset_btn">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>250</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>30</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>link1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>60</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>link2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>90</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>link3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label1">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>160</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>joint1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>190</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>join2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label3">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>220</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>joint3</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>290</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>320</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>290</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>320</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>y</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>360</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
